docs/how-to/enable-nts.md

```diff
@@ -0,0 +1,43 @@
+# How to: Enable NTS (Network Time Security)
+
+Network Time Security (NTS) is a mechanism that uses Transport Layer Security
+(TLS) and Authenticated Encryption with Associated Data (AEAD) to provide 
+cryptographic security for the client-server mode of the Network Time Protocol
+(NTP).
+
+When a TLS certificate is provided to the Chrony charm, Chrony will 
+automatically enable NTS, allowing NTS-capable NTP clients to securely
+communicate with the Chrony charm. 
+You can use one of the following options to configure the TLS certificate
+used for NTS.
+
+## `tls-certificates` integration
+
+The Chrony charm can integrate with any charm that provides `tls-certificates` 
+integration, such as the [`lego`](https://charmhub.io/lego) or 
+[`self-signed-certificates`](https://charmhub.io/self-signed-certificates) 
+charms. 
+Refer to the respective `tls-certificates` provider charm's documentation 
+for setup instructions. 
+Once set up, you can integrate the `tls-certificates` provider charm with 
+the Chrony charm:
```

Comment:
I think it would be good to indicate that you're showing an example below

Reply:
Okay, I rephrased it a little. What do you think?

Reply:
It looks good to me :)